src/lib/GlobalEventListener.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
this can live within demosplan-core, as it is
- a custom nongeneric function
- used on elements that are unique to demosplan-core

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/71/commits/d90277fdb90ee4349783e5c1e51137121a6e26c6
New PR in demosplan-core: https://github.com/demos-europe/demosplan-core/pull/666